tests/transform/test_create_segment_map.py

```diff
@@ -0,0 +1,111 @@
+                "first_msg_lon": 4.8533333333333335,
+                "first_msg_course": None,
+                "first_msg_speed": 0.0,
+                "last_msg_timestamp": datetime(2024, 4, 12, 12).timestamp(),
+                "last_msg_lat": 45.785,
+                "last_msg_lon": 4.8533333333333335,
+                "last_msg_course": None,
+                "last_msg_speed": 0.0,
+                "msg_count": 1,
+                "identities": [],
+                "destinations": [],
+            },
+            {
+                "frag_id": "226010660-2024-04-13T00:00:00.000000Z-1",
+                "ssvid": "226010660",
+                "timestamp": datetime(2024, 4, 13, 0).timestamp(),
+                "first_msg_timestamp": datetime(2024, 4, 13, 0).timestamp(),
+                "first_msg_lat": 45.785,
+                "first_msg_lon": 4.8533333333333335,
+                "first_msg_course": None,
+                "first_msg_speed": 0.0,
+                "last_msg_timestamp": datetime(2024, 4, 13, 0).timestamp(),
+                "last_msg_lat": 45.785,
+                "last_msg_lon": 4.8533333333333335,
+                "last_msg_course": None,
+                "last_msg_speed": 0.0,
+                "msg_count": 1,
+                "identities": [],
+                "destinations": [],
+            },
+            {
+                "frag_id": "226010660-2024-04-15T00:00:00.000000Z-1",
+                "ssvid": "226010660",
+                "timestamp": datetime(2024, 4, 15).timestamp(),
+                "first_msg_timestamp": datetime(2024, 4, 15).timestamp(),
+                "first_msg_lat": 45.785,
+                "first_msg_lon": 4.8533333333333335,
+                "first_msg_course": None,
+                "first_msg_speed": 0.0,
+                "last_msg_timestamp": datetime(2024, 4, 15).timestamp(),
+                "last_msg_lat": 45.785,
+                "last_msg_lon": 4.8533333333333335,
+                "last_msg_course": None,
+                "last_msg_speed": 0.0,
+                "msg_count": 1,
+                "identities": [],
+                "destinations": [],
+            },
+        ],
+    )
+]
+
+MERGE_PARAMS = dict(buffer_hours=0.5)
+
+
+OUTPUT = [
+    {
+        "seg_id": "226010660-2024-04-12T12:00:00.000000Z-1",
+        "date": date(2024, 4, 12),
+        "frag_id": "226010660-2024-04-12T12:00:00.000000Z-1",
+        'ssvid': '226010660',
+    },
+    {
+        "seg_id": "226010660-2024-04-12T12:00:00.000000Z-1",
+        "date": date(2024, 4, 13),
+        "frag_id": "226010660-2024-04-13T00:00:00.000000Z-1",
+        'ssvid': '226010660',
+    },
+    {
+        "seg_id": "226010660-2024-04-15T00:00:00.000000Z-1",
+        "date": date(2024, 4, 15),
+        "frag_id": "226010660-2024-04-15T00:00:00.000000Z-1",
+        'ssvid': '226010660'
+    },
+]
+
+
+def test_create_segment_map():
+    op = CreateSegmentMap()  # No params
+
+    op = CreateSegmentMap(MERGE_PARAMS)
```

Comment:
Why both?

Reply:
If it's testing 2 different code paths, I would prefer to have 2 separate tests, one that just constructs the segment map with no params and ensures that doesn't blow up, and one that does the actual testing you did here.

Reply:
Done